src/app/views/dashboard/main/overview/overview.html

```diff
@@ -393,6 +393,80 @@ <h2 :class="{ 'section-header': Screen.isDesktop }">
 					</div>
 				</template>
 
+				<!--
+					Manage Collaborations
+				-->
```

Comment:
merge this into the normal "Your Games" list and instead just show a tag underneath that says "Collaborator".

Allow them to stop collaborating when they're managing the game (in the game overview), not from this screen.